certified-connectors/nimflow/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Nimflow Connector
```

Comment:
Please update readme.md file as per the template in the below path. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
done